packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/create/iti55-envelope.ts

```diff
@@ -177,10 +177,7 @@ function createSoapBodyContent({
                   [`${prefix}livingSubjectName`]: {
                     [`${prefix}value`]: {
                       [`${prefix}family`]: name.family,
-                      ...name.given?.reduce((acc: { [key: string]: string }, givenName: string) => {
-                        acc[`${prefix}given`] = givenName;
-                        return acc;
-                      }, {}),
+                      [`${prefix}given`]: name.given,
```

Comment:
use of accumulator here meant that if we had multiple given names, we would always overwrite and then only use the last one

Reply:
we probably have been losing patient data on this. 
fastXml parser by default if given an array will construct multiple "given" sub tags, so now we just do that

Reply:
is this tested to have a positive impact? do we know that after this change we will still be creating valid requests?